lib/mail/parsers/address_lists_parser.rb

```diff
@@ -31992,7 +31992,6 @@ def self.parse(data)
 
       
 begin
-	testEof = false
```

Comment:
`lib/mail/parsers/*_parser.rb` are generated from `.rl` files by Ragel—this is boilerplate generated Ruby code that we can't change. Hence the `$VERBOSE = nil` wrapping the parser definition, suppressing warnings 😑

Reply:
Oh... we do see the warnings running gems that require mail, but it seems that correcting this should be done elsewhere. Sorry for the noise.